modules/ROOT/pages/installation/sso.adoc

```diff
@@ -0,0 +1,271 @@
+| `false`
+
+| `app.sso.configurations.<CONFIG>.idp-type`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_IDP_TYPE`
+| Type of the Identity Provider (IdP). Currently only `ENTRA_ID`, `GOOGLE_IDENTITY` and `OKTA` are supported.
+| `ENTRA_ID`
+
+| `app.sso.configurations.<CONFIG>.display-name`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_DISPLAY_NAME`
+| Display name of the login option on the NOM login page.
+| `Login with Entra ID`
+
+| `app.sso.configurations.<CONFIG>.admin-check`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_ADMIN_CHECK`
+| SpEL expression to check if the user should be assigned the NOM ADMIN role in NOM.
+See xref:_role_assignment[Role assignment] for details.
+| `['groups']?.contains('aaa63d30-70c3-4181-8c19-1b58e613f55b')`
+
+| `app.sso.configurations.<CONFIG>.read-only-user-check`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_READ_ONLY_USER_CHECK`
+| SpEL expression to check if the user should be assigned the NOM READ_ONLY_USER role in NOM.
+See xref:_role_assignment[Role assignment] for details.
+| `['groups']?.contains('bbb2c5c5-a181-458a-a933-8ea7339d1310')`
+
+| `app.sso.configurations.<CONFIG>.username-claim`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_USERNAME_CLAIM`
+| SpEL expression to override the default field for the username in the token claims.
+| `['unique_name']`
+
+| `app.sso.configurations.<CONFIG>.email-claim`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_EMAIL_CLAIM`
+| SpEL expression to override the default field for the email in the token claims.
+| `['verified_email']`
+
+| `app.sso.configurations.<CONFIG>.firstname-claim`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_FIRSTNAME_CLAIM`
+| SpEL expression to override the default field for the first name in the token claims.
+| `['given_name']`
+
+| `app.sso.configurations.<CONFIG>.lastname-claim`
+| `APP_SSO_CONFIGURATIONS_<CONFIG>_LASTNAME_CLAIM`
+| SpEL expression to override the default field for the last name in the token claims.
+| `['family_name']`
+|===
+
+[#_role_assignment]
+==== Role assignment
+
+The `admin-check` and `read-only-user-check` options are https://docs.spring.io/spring-framework/reference/core/expressions/language-ref.html[SpEL expressions] that are evaluated against the claims in the access token obtained from the IdP.
+
+The claims object is available as the root object in the expression context.
+
+The expressions should evaluate to `true` or `false`.
+If the `admin-check` expression evaluates to `true`, the user is assigned the NOM Admin role in NOM.
+Otherwise, the read-only-user-check expression is evaluated.
+If it evaluates to `true`, the user is assigned the read-only role in NOM.
+If both expressions evaluate to `false`, the user is denied access to NOM.
+
+The permissions of the read-only role include:
+
+* Viewing DBMS status
+* Viewing DBMS metrics
+* Viewing and downloading DBMS query logs
+* Viewing agents and their details
+* Viewing the Upgrades page
+* Read-only access to the DBMS Security page
```

Comment:
Maybe better to remove this list and reference https://neo4j.com/docs/ops-manager/current/nom-user-management/#_roles_of_nom_users